charts/peacock/values.yaml

```diff
@@ -17,9 +17,37 @@ webhookSecret: ""
 # Token for authenticating the message webhook
 webhookToken: ""
 
+# The connection string used for the database
+mongodbConnectionString: ""
```

Comment:
Let's describe that this is an override + How to use it?

Worth putting a chart readme?